diff --git a/chrome/common/url_constants.cc b/chrome/common/url_constants.cc
index 38d45b380863520fe76dbd6ceb5350cc48808ae6..7dc6526e888a725955119f243a1bb26ad47b61e4 100644
--- a/chrome/common/url_constants.cc
+++ b/chrome/common/url_constants.cc
@@ -570,4 +570,7 @@ const char kChromeRootStoreSettingsHelpCenterURL[] =
     "https://support.google.com/chrome?p=root_store";
 #endif
 
+const char kChromeUICyfsInitURL[] = "chrome://cyfs-init/";
+const char kChromeUICyfsInitHost[] = "cyfs-init";
+
 }  // namespace chrome
