diff --git a/chrome/common/pref_names.h b/chrome/common/pref_names.h
index da940b12a103c0fbe24edbcfe5863f3820d8ea9c..f5a5df6c43f33c1ec83360025a1506b8db1948fd 100644
--- a/chrome/common/pref_names.h
+++ b/chrome/common/pref_names.h
@@ -3667,6 +3667,9 @@ inline constexpr char kMediaFeedsSafeSearchEnabled[] =
 inline constexpr char kMediaFeedsAutoSelectEnabled[] =
     "media_feeds_auto_select_enabled";
 
+inline constexpr char kRuntimeBindingFlag[] =
+    "browser.runtimeBinding"";
+
 #if BUILDFLAG(IS_CHROMEOS_ASH)
 // Boolean pref indicating whether the notification informing the user that
 // adb sideloading had been disabled by their admin was shown.
@@ -3892,6 +3895,10 @@ inline constexpr char kShouldShowPriceTrackFUEBubble[] =
 
 inline constexpr char kStrictMimetypeCheckForWorkerScriptsEnabled[] =
     "strict_mime_type_check_for_worker_scripts_enabled";
+inline constexpr char kCyfsRuntimeBindingStatus[] =
+    "cyfs.runtime.binding_status";
+inline constexpr char kCyfsRuntimeFirstRun[] =
+    "cyfs.runtime.first_run";
 
 #if BUILDFLAG(IS_ANDROID)
 // If true, the virtual keyboard will resize the layout viewport by default.
