diff --git a/chrome/chrome_proxy/chrome_proxy_main_win.cc b/chrome/chrome_proxy/chrome_proxy_main_win.cc
index 4ba7702bfa283c5bc376dacbdcddd98c71b7ba04..c5b5b8b6898e33a097a56462f625c7b92644a287 100644
--- a/chrome/chrome_proxy/chrome_proxy_main_win.cc
+++ b/chrome/chrome_proxy/chrome_proxy_main_win.cc
@@ -14,7 +14,7 @@
 namespace {
 
 constexpr base::FilePath::CharType kChromeExecutable[] =
-    FILE_PATH_LITERAL("chrome.exe");
+    FILE_PATH_LITERAL("Kalama.exe");
 
 constexpr base::FilePath::CharType kChromeProxyExecutable[] =
     FILE_PATH_LITERAL("chrome_proxy.exe");
