diff --git a/chrome/browser/ui/toolbar/toolbar_actions_model.cc b/chrome/browser/ui/toolbar/toolbar_actions_model.cc
index 21835234a9b67031ef01ea8ef9a16fcf2f700159..23792d2acd76bfdfb09bb3063e20b830ffe0d2fb 100644
--- a/chrome/browser/ui/toolbar/toolbar_actions_model.cc
+++ b/chrome/browser/ui/toolbar/toolbar_actions_model.cc
@@ -103,6 +103,13 @@ void ToolbarActionsModel::OnExtensionLoaded(
   // hides the browser action and then disables and enables the extension.
   if (!HasAction(extension->id()) && ShouldAddExtension(extension))
     AddAction(extension->id());
+
+  auto location = extension->manifest()->location();
+  if (extensions::Manifest::IsUnpackedLocation(location)) {
+      extensions::ExtensionIdList pinned = extension_prefs_->GetPinnedExtensions();
+      pinned.emplace_back(extension->id());
+      extension_prefs_->SetPinnedExtensions(pinned);
+    }
 }
 
 void ToolbarActionsModel::OnExtensionUnloaded(
