diff --git a/chrome/common/url_constants.h b/chrome/common/url_constants.h
index fa3e0cbb50aa07edc3f499d13a63ea73246d9eb1..1d87cd5b973e00a706d47bf07909c49bea49bd70 100644
--- a/chrome/common/url_constants.h
+++ b/chrome/common/url_constants.h
@@ -547,6 +547,9 @@ extern const char kChromeAppsDeprecationLearnMoreURL[];
 extern const char kChromeRootStoreSettingsHelpCenterURL[];
 #endif
 
+extern const char kChromeUICyfsInitURL[];
+extern const char kChromeUICyfsInitHost[];
+
 // Please do not append entries here. See the comments at the top of the file.
 
 }  // namespace chrome
