diff --git a/chrome/common/pref_names.h b/chrome/common/pref_names.h
index 1428bb8d1b56869c67245e9b971b79fb5717d935..66cf036010037193b3cf0431272331fa1f2f95bf 100644
--- a/chrome/common/pref_names.h
+++ b/chrome/common/pref_names.h
@@ -2868,6 +2868,12 @@ inline constexpr char kKnownUserParentAccessCodeConfig[] =
     "child_user.parent_access_code.config";
 #endif  // BUILDFLAG(IS_CHROMEOS_ASH)
 
+// Cyfs pref
+inline constexpr char kRuntimeBindingFlag[] = "browser.runtimeBinding";
+inline constexpr char kCyfsRuntimeBindingStatus[] = "cyfs.runtime.binding_status";
+inline constexpr char kCyfsRuntimeFirstRun[] = "cyfs.runtime.first_run";
+
+
 // String which specifies where to store the disk cache.
 inline constexpr char kDiskCacheDir[] = "browser.disk_cache_dir";
 // Pref name for the policy specifying the maximal cache size.
