diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 9da0c57372d3de3d47cc745602eb6f11a394ce5e..7605d7c10a84516e801ff63e985061ccd8349b32 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -357,6 +357,8 @@ static_library("ui") {
     "zoom/chrome_zoom_level_otr_delegate.h",
     "zoom/chrome_zoom_level_prefs.cc",
     "zoom/chrome_zoom_level_prefs.h",
+    "webui/cyfs_init_ui.cc",
+    "webui/cyfs_init_ui.h",
   ]
 
   if (enable_session_service) {
