diff --git a/chrome/browser/extensions/api/runtime/chrome_runtime_api_delegate.cc b/chrome/browser/extensions/api/runtime/chrome_runtime_api_delegate.cc
index fcc035ef0423071d540f8abb39147cd0ffc8d452..e7d561cc5af3506a1d36f8710b08befa1db58bfb 100644
--- a/chrome/browser/extensions/api/runtime/chrome_runtime_api_delegate.cc
+++ b/chrome/browser/extensions/api/runtime/chrome_runtime_api_delegate.cc
@@ -26,6 +26,7 @@
 #include "chrome/browser/ui/browser_finder.h"
 #include "chrome/browser/ui/browser_navigator.h"
 #include "chrome/browser/ui/browser_navigator_params.h"
+#include "chrome/browser/lifetime/application_lifetime.h"
 #include "components/update_client/update_query_params.h"
 #include "content/public/browser/notification_service.h"
 #include "extensions/browser/extension_system.h"
@@ -333,9 +334,8 @@ bool ChromeRuntimeAPIDelegate::RestartDevice(std::string* error_message) {
     return true;
   }
 #endif
-
-  *error_message = "Function available only for ChromeOS kiosk mode.";
-  return false;
+  chrome::AttemptRestart();
+  return true;
 }
 
 bool ChromeRuntimeAPIDelegate::OpenOptionsPage(
