diff --git a/chrome/version.gni b/chrome/version.gni
index 3cb46611efd5698e8a87b1f3dce94b0b4358fba0..3d0a49d8fea9c2ac696883f541a558c81527a47f 100644
--- a/chrome/version.gni
+++ b/chrome/version.gni
@@ -23,6 +23,7 @@ _version_dictionary_template = "full = \"@MAJOR@.@MINOR@.@BUILD@.@PATCH@\" " +
 
 # The file containing the Chrome version number.
 chrome_version_file = "//chrome/VERSION"
+kalama_version_file = "//chrome/KALAMA_VERSION"
 
 _script_arguments = []
 
@@ -125,6 +126,21 @@ _result = exec_script("//build/util/version.py",
 # Full version. For example "45.0.12321.0"
 chrome_version_full = _result.full
 
+kalama_version_full =
+    exec_script(
+        "//build/util/version.py",
+        [
+          "-f",
+          rebase_path(chrome_version_file, root_build_dir),
+          "-f",
+          rebase_path(kalama_version_file, root_build_dir),
+          "-t",
+          "@KALAMA_MAJOR@.@KALAMA_MINOR@.@KALAMA_NIGHTLY@.@KALAMA_BUILD@",
+        ],
+        "trim string",
+        [chrome_version_file, kalama_version_file]
+      )
+
 # The constituent parts of the full version.
 chrome_version_major = _result.major
 chrome_version_minor = _result.minor
