diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 31bf3d854d9a672e535f747fb8734d66ac0c37e9..71be2b061847ff32e9d3e14157ba957608451535 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -45,6 +45,7 @@ import("//third_party/widevine/cdm/widevine.gni")
 import("//tools/grit/grit_rule.gni")
 import("//ui/base/ui_features.gni")
 
+
 # //build/config/android/rules.gni imports //tools/grit/grit_rule.gni, which
 # produces a conflict for the "grit" template so we have to only include one.
 if (is_android) {
@@ -1887,6 +1888,8 @@ static_library("browser") {
     "webid/federated_identity_permission_context.h",
     "webid/federated_identity_permission_context_factory.cc",
     "webid/federated_identity_permission_context_factory.h",
+    "cyfs_init/cyfs_init_service_factory.h",
+    "cyfs_init/cyfs_init_service_factory.cc"
   ]
 
   if (!is_android) {
@@ -2632,6 +2635,8 @@ static_library("browser") {
     ]
   }
 
+  deps += ["//components/cyfs_init"]
+
   if (is_android) {
     sources += [
       "accessibility/accessibility_prefs/android/accessibility_prefs_controller.cc",
