diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 619f741f150975e1a93a32783f9bbfe78e9bee15..cd229970b8e558bc741f6e48da9d4c5afb737acc 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -413,7 +413,7 @@ void ToolbarView::Init() {
     toolbar_divider_->SetPreferredSize(
         gfx::Size(kToolbarDividerWidth, kToolbarDividerHeight));
   }
-
+/*
   if (IsChromeLabsEnabled()) {
     chrome_labs_model_ = std::make_unique<ChromeLabsModel>();
     UpdateChromeLabsNewBadgePrefs(browser_->profile(),
@@ -433,7 +433,7 @@ void ToolbarView::Init() {
       chrome_labs_button_->SetVisible(show_chrome_labs_button_.GetValue());
     }
   }
-
+*/
   if (!IsCrosBatterySaverAvailable()) {
     battery_saver_button_ = container_view_->AddChildView(
         std::make_unique<BatterySaverButton>(browser_view_));
