diff --git a/chrome/browser/extensions/installed_loader.h b/chrome/browser/extensions/installed_loader.h
index 3a24cc3966f7aefd45c9f22225404439cfc9fcc9..d179e52c15f78d7afe0e130aa3ea6edf9503d394 100644
--- a/chrome/browser/extensions/installed_loader.h
+++ b/chrome/browser/extensions/installed_loader.h
@@ -6,6 +6,8 @@
 #define CHROME_BROWSER_EXTENSIONS_INSTALLED_LOADER_H_
 
 #include <set>
+#include <vector>
+#include <string>
 
 #include "base/files/file_path.h"
 #include "base/memory/raw_ptr.h"
@@ -68,6 +70,10 @@ class InstalledLoader {
   void RecordExtensionsMetrics(Profile* profile,
                                bool log_user_profile_histograms);
 
+  std::vector<base::FilePath> GetDefaultInstalledExtensions();
+
+  std::vector<std::string> GetExtensionBlacklist();
+
   raw_ptr<ExtensionService> extension_service_;
   raw_ptr<ExtensionRegistry> extension_registry_;
 
