diff --git a/chrome/browser/extensions/extension_service.cc b/chrome/browser/extensions/extension_service.cc
index bbbf5031201c3e37a9cdefad3c4887fc2a205cb9..c4c5e8d993786c07490efe51252b30fe8d4fe225 100644
--- a/chrome/browser/extensions/extension_service.cc
+++ b/chrome/browser/extensions/extension_service.cc
@@ -82,6 +82,7 @@
 #include "content/public/browser/notification_service.h"
 #include "content/public/browser/render_process_host.h"
 #include "content/public/browser/storage_partition.h"
+#include "extensions/browser/pref_names.h"
 #include "extensions/browser/blocklist_extension_prefs.h"
 #include "extensions/browser/blocklist_state.h"
 #include "extensions/browser/disable_reason.h"
@@ -809,6 +810,23 @@ bool ExtensionService::UninstallExtension(
   // Callers should not send us nonexistent extensions.
   CHECK(extension.get());
 
+  {
+    base::FilePath extension_path = extension->path();
+    VLOG(1) << "uninstall extension, path = " << extension_path
+          << " id = " << extension->id();
+    std::string extension_path_name = extension_path.BaseName().AsUTF8Unsafe();
+    {
+      PrefService* prefs = profile()->GetPrefs();
+      ScopedListPrefUpdate update(prefs, pref_names::KDefaultExtensionBlackList);
+      VLOG(1) << "Add extension " << extension_path_name << " to default extension black list";
+      update->Append(extension_path_name);
+    }
+    // delete extension path
+    if (base::PathExists(extension_path)) {
+      base::DeletePathRecursively(extension_path);
+    }
+  }
+
   ManagementPolicy* by_policy = system_->management_policy();
   // Policy change which triggers an uninstall will always set
   // |external_uninstall| to true so this is the only way to uninstall
