diff --git a/chrome/installer/setup/setup_main.cc b/chrome/installer/setup/setup_main.cc
index 0000c1392d4a75ceb856414b1243ba46737877f8..b24d8b70fd87575c3af957eb4472ef6ea98b3c59 100644
--- a/chrome/installer/setup/setup_main.cc
+++ b/chrome/installer/setup/setup_main.cc
@@ -19,6 +19,7 @@
 
 #include "base/at_exit.h"
 #include "base/command_line.h"
+#include "base/debug/stack_trace.h"
 #include "base/file_version_info.h"
 #include "base/files/file_path.h"
 #include "base/files/file_util.h"
@@ -744,6 +745,7 @@ installer::InstallStatus UninstallProducts(InstallationState& original_state,
       current_version,
   };
 
+  VLOG(1) << __FUNCTION__ << " cmd " << cmd_line.GetCommandLineString();
   install_status = UninstallProduct(modify_params, remove_all, force, cmd_line);
 
   installer::CleanUpInstallationDirectoryAfterUninstall(
@@ -781,6 +783,7 @@ installer::InstallStatus InstallProducts(InstallationState& original_state,
                                          const InitialPreferences& prefs,
                                          InstallerState* installer_state,
                                          base::FilePath* installer_directory) {
+  VLOG(1) << __FUNCTION__ << " cmd " << cmd_line.GetArgumentsString();
   DCHECK(installer_state);
   installer::InstallStatus install_status = installer::UNKNOWN_STATUS;
   installer::ArchiveType archive_type = installer::UNKNOWN_ARCHIVE_TYPE;
@@ -947,6 +950,7 @@ bool HandleNonInstallCmdLineOptions(installer::ModifyParams& modify_params,
                                     const base::CommandLine& cmd_line,
                                     const InitialPreferences& prefs,
                                     int* exit_code) {
+  VLOG(1) << __FUNCTION__ << " cmd: " << cmd_line.GetArgumentsString();
   installer::InstallerState* installer_state =
       &(*modify_params.installer_state);
   installer::InstallationState* original_state =
@@ -1265,6 +1269,7 @@ InstallStatus InstallProductsHelper(InstallationState& original_state,
                                     InstallerState& installer_state,
                                     base::FilePath* installer_directory,
                                     ArchiveType* archive_type) {
+  VLOG(1) << __FUNCTION__;
   DCHECK(archive_type);
   const bool system_install = installer_state.system_install();
   InstallStatus install_status = UNKNOWN_STATUS;
@@ -1387,6 +1392,8 @@ InstallStatus InstallProductsHelper(InstallationState& original_state,
 
       const base::Version current_version(
           installer_state.GetCurrentVersion(original_state));
+      VLOG(1) << "Current version: " << current_version.GetString()
+              << "Installer version " << current_version.GetString();
       InstallParams install_params = {
           installer_state,  original_state,       setup_exe,
           current_version,  uncompressed_archive, src_path,
@@ -1432,7 +1439,7 @@ InstallStatus InstallProductsHelper(InstallationState& original_state,
       if (install_status == FIRST_INSTALL_SUCCESS) {
         VLOG(1) << "First install successful.";
         // We never want to launch Chrome in system level install mode.
-        bool do_not_launch_chrome = false;
+        bool do_not_launch_chrome = true;
         prefs.GetBool(initial_preferences::kDoNotLaunchChrome,
                       &do_not_launch_chrome);
         if (!system_install && !do_not_launch_chrome)
@@ -1624,9 +1631,11 @@ int SetupMain() {
   // true path to the program at position 0.
   base::FilePath setup_exe;
   base::PathService::Get(base::FILE_EXE, &setup_exe);
+  VLOG(1) << "Using setup exe " << setup_exe.AsUTF8Unsafe();
 
   const base::Version current_version(
       installer_state.GetCurrentVersion(original_state));
+  VLOG(1) << "Current version: " << current_version.GetString();
   installer::ModifyParams modify_params = {
       installer_state,
       original_state,
@@ -1679,9 +1688,11 @@ int SetupMain() {
   installer::InstallStatus install_status = installer::UNKNOWN_STATUS;
   // If --uninstall option is given, uninstall the identified product(s)
   if (is_uninstall) {
+    VLOG(1) << __FUNCTION__ << " UninstallProducts";
     install_status =
         UninstallProducts(original_state, installer_state, setup_exe, cmd_line);
   } else {
+    VLOG(1) << __FUNCTION__ << " InstallProducts";
     // If --uninstall option is not specified, we assume it is install case.
     install_status = InstallProducts(original_state, setup_exe, cmd_line, prefs,
                                      &installer_state, &installer_directory);
