diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
index 6f989cff781d71e2ea534d5592bf3d6fcaa0e95f..c557d2ed7fe2a14544603bbff026334e9961d4a4 100644
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -1357,7 +1357,7 @@ base::FilePath GetModulePath(base::WStringPiece module_name) {
   // executable's directory and return the path if it can be read. This is the
   // expected location of modules for proper installs.
   const base::FilePath module_path =
-      exe_dir.AppendASCII(chrome::kChromeVersion).Append(module_name);
+      exe_dir.AppendASCII(chrome::kProductVersion).Append(module_name);
   if (base::PathExists(module_path))
     return module_path;
 
