diff --git a/chrome/tools/build/win/create_installer_archive.py b/chrome/tools/build/win/create_installer_archive.py
index ba75899c40db87a3ddb138912855cb6e1bbca45d..135def8367cb93a2ef58931ba515868d3b2bd5b2 100755
--- a/chrome/tools/build/win/create_installer_archive.py
+++ b/chrome/tools/build/win/create_installer_archive.py
@@ -41,6 +41,7 @@ SETUP_EXEC = "setup.exe"
 SETUP_PATCH_FILE_PREFIX = "setup_patch"
 TEMP_ARCHIVE_DIR = "temp_installer_archive"
 VERSION_FILE = "VERSION"
+KALAMA_VERSION_FILE = "KALAMA_VERSION"
 
 
 g_archive_inputs = []
@@ -56,17 +57,17 @@ def BuildVersion():
   patch = 0
   # The version file is located at the directory ${CHROMIUM_SRC}/chrome.
   version_file_path = os.path.join(
-      os.path.dirname(__file__), '../../..', VERSION_FILE)
+      os.path.dirname(__file__), '../../..', KALAMA_VERSION_FILE)
   for line in open(version_file_path, 'r'):
     line = line.rstrip()
-    if line.startswith('MAJOR='):
-      major = line[6:]
-    elif line.startswith('MINOR='):
-      minor = line[6:]
-    elif line.startswith('BUILD='):
-      build = line[6:]
-    elif line.startswith('PATCH='):
-      patch = line[6:]
+    if line.startswith('KALAMA_MAJOR='):
+      major = line[13:]
+    elif line.startswith('KALAMA_MINOR='):
+      minor = line[13:]
+    elif line.startswith('KALAMA_NIGHTLY='):
+      build = line[15:]
+    elif line.startswith('KALAMA_BUILD='):
+      patch = line[13:]
   return '%s.%s.%s.%s' % (major, minor, build, patch)
 
 
@@ -484,6 +485,7 @@ def ParseDLLsFromDeps(build_dir, runtime_deps_file):
 def DoComponentBuildTasks(staging_dir, build_dir, target_arch,
                           setup_runtime_deps, chrome_runtime_deps,
                           current_version):
+  print("version %s" % current_version)
   # Get the required directories for the upcoming operations.
   chrome_dir = os.path.join(staging_dir, CHROME_DIR)
   version_dir = os.path.join(chrome_dir, current_version)
