diff --git a/chrome/common/webui_url_constants.h b/chrome/common/webui_url_constants.h
index ce0e4a432654244344c7f12a1a8e36f0bce2a616..7f0403446566400531245d6fcf352e295f8abe02 100644
--- a/chrome/common/webui_url_constants.h
+++ b/chrome/common/webui_url_constants.h
@@ -38,6 +38,8 @@ extern const char kChromeUIAppIconHost[];
 extern const char kChromeUIAppIconURL[];
 extern const char kChromeUIAppLauncherPageHost[];
 extern const char kChromeUIAppsURL[];
+extern const char kCyfsBrowserURL[];
+extern const char kCyfsBrowserGuideURL[];
 extern const char kChromeUIAppsWithDeprecationDialogURL[];
 extern const char kChromeUIAppsWithForceInstalledDeprecationDialogURL[];
 extern const char kChromeUIAutofillInternalsHost[];
