diff --git a/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc b/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
index 2c54740738d1fbaf977eb150e91428596ff9ed94..5aa779727972b6427bd2d1aed2f9bf59f1f70a39 100644
--- a/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
+++ b/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
@@ -350,9 +350,25 @@ void AddAboutStrings(content::WebUIDataSource* html_source, Profile* profile) {
       l10n_util::GetStringUTF16(IDS_SETTINGS_UPGRADE_UP_TO_DATE));
 #endif
 
-  std::u16string browser_version = VersionUI::GetAnnotatedVersionStringForUi();
+  std::string channel_name =
+      chrome::GetChannelName(chrome::WithExtendedStable(true));
+  std::string cyfs_browser_version = chrome::GetCYFSBrowserVersion();
+  std::string cyfs_version = cyfs_browser_version + std::string(" Chromium: ") + std::string(version_info::GetVersionNumber());
+
+  std::u16string browser_version = l10n_util::GetStringFUTF16(
+        IDS_SETTINGS_ABOUT_PAGE_BROWSER_VERSION,
+        base::UTF8ToUTF16(cyfs_version),
+        l10n_util::GetStringUTF16(version_info::IsOfficialBuild()
+                                      ? IDS_VERSION_UI_OFFICIAL
+                                      : IDS_VERSION_UI_UNOFFICIAL),
+        base::UTF8ToUTF16(channel_name),
+        l10n_util::GetStringUTF16(VersionUI::VersionProcessorVariation()));
 
   html_source->AddString("aboutBrowserVersion", browser_version);
+  std::string runtime_version_string = chrome::GetCYFSRuntimeVersion();
+  std::string runtime_version = std::string("Module Version: ") + runtime_version_string;
+  html_source->AddString("aboutRuntimeVersion", runtime_version);
+
   html_source->AddString(
       "aboutProductCopyright",
       base::i18n::MessageFormatter::FormatWithNumberedArgs(
