diff --git a/components/os_crypt/sync/keychain_password_mac.mm b/components/os_crypt/sync/keychain_password_mac.mm
index f6269925af68ce3ef9fb684bf4a1662874becae9..c133ca478c5ca2f3812f2cfb21635f6b5dec4a52 100644
--- a/components/os_crypt/sync/keychain_password_mac.mm
+++ b/components/os_crypt/sync/keychain_password_mac.mm
@@ -31,8 +31,8 @@
 const char kDefaultServiceName[] = "Chrome Safe Storage";
 const char kDefaultAccountName[] = "Chrome";
 #else
-const char kDefaultServiceName[] = "Chromium Safe Storage";
-const char kDefaultAccountName[] = "Chromium";
+const char kDefaultServiceName[] = "Kalama Safe Storage";
+const char kDefaultAccountName[] = "Kalama";
 #endif
 
 // Generates a random password and adds it to the Keychain.  The added password
