diff --git a/content/app/content_main_runner_impl.cc b/content/app/content_main_runner_impl.cc
index 11dcd5eaabaaba0e297ecb992d1290b3adaff1b1..04931f47ba581dfd4c71a201e83fd785dd229922 100644
--- a/content/app/content_main_runner_impl.cc
+++ b/content/app/content_main_runner_impl.cc
@@ -1148,6 +1148,13 @@ int ContentMainRunnerImpl::RunBrowser(MainFunctionParams main_params,
   if (is_browser_main_loop_started_)
     return -1;
 
+  #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
+  if (!is_runtime_initialized_) {
+    monitor_runtime_work_ = std::make_unique<MonitorRuntimeWork>();
+    is_runtime_initialized_ = true;
+  }
+  #endif
+
   if (!base::CommandLine::ForCurrentProcess()->HasSwitch(
           switches::kSingleProcess)) {
     mojo::SyncCallRestrictions::DisableSyncCallInterrupts();
