diff --git a/extensions/common/extension.cc b/extensions/common/extension.cc
index 75a110cbab5b26c2c53799044498e9b17a66fbd4..d50e98d00550720016ed8e03c0196d41b37268d4 100644
--- a/extensions/common/extension.cc
+++ b/extensions/common/extension.cc
@@ -51,6 +51,13 @@ namespace keys = manifest_keys;
 namespace values = manifest_values;
 namespace errors = manifest_errors;
 
+const base::FilePath::CharType* kDefaultExtensionNames[]= {
+    FILE_PATH_LITERAL("metamask-chrome"),
+    FILE_PATH_LITERAL("CyberChat"),
+    FILE_PATH_LITERAL("DMC"),
+};
+const int kDefaultExtensionNum = 3;
+
 namespace {
 
 constexpr int kMinimumSupportedManifestVersion = 2;
