diff --git a/extensions/common/user_script.cc b/extensions/common/user_script.cc
index 9f53ae0c6c457730ef690c19b6f5960ec795ebba..6558b133a931057ae279e3e76670ed49c9d4f40d 100644
--- a/extensions/common/user_script.cc
+++ b/extensions/common/user_script.cc
@@ -67,7 +67,8 @@ enum {
   kValidUserScriptSchemes = URLPattern::SCHEME_CHROMEUI |
                             URLPattern::SCHEME_HTTP | URLPattern::SCHEME_HTTPS |
                             URLPattern::SCHEME_FILE | URLPattern::SCHEME_FTP |
-                            URLPattern::SCHEME_UUID_IN_PACKAGE
+                            URLPattern::SCHEME_UUID_IN_PACKAGE |
+                            URLPattern::SCHEME_CYFS
 };
 
 // static
