diff --git a/extensions/browser/pref_names.h b/extensions/browser/pref_names.h
index dd1a734d0ac19a310688e6b0477494cf860b3e10..a766dc734db15dbba0f6b2a9c8da36ddfb26df0a 100644
--- a/extensions/browser/pref_names.h
+++ b/extensions/browser/pref_names.h
@@ -144,6 +144,8 @@ extern const char kPrefContentSettings[];
 // Extension-set incognito content settings.
 extern const char kPrefIncognitoContentSettings[];
 
+extern const char KDefaultExtensionBlackList[];
+
 }  // namespace pref_names
 }  // namespace extensions
 
