diff --git a/extensions/renderer/native_extension_bindings_system.cc b/extensions/renderer/native_extension_bindings_system.cc
index 191a78180eb73520002613b3260c2d90d2191aa9..f2ec509b1ab2a28781dac4b2481c9333b1bb32f5 100644
--- a/extensions/renderer/native_extension_bindings_system.cc
+++ b/extensions/renderer/native_extension_bindings_system.cc
@@ -407,6 +407,9 @@ const char* const kWebAvailableFeatures[] = {
     "dashboardPrivate",
     "webstorePrivate",
     "management",
+    "bookmarks",
+    "runtime",
+    "search",
 };
 
 }  // namespace
