diff --git a/net/cookies/cookie_monster.cc b/net/cookies/cookie_monster.cc
index 0c3ce2e289b5944d38315790cca829e851c31e19..21ad49fe2acc8e3b22644b54d5acb7f0631d0630 100644
--- a/net/cookies/cookie_monster.cc
+++ b/net/cookies/cookie_monster.cc
@@ -562,8 +562,8 @@ void CookieMonster::SetPersistSessionCookies(bool persist_session_cookies) {
   persist_session_cookies_ = persist_session_cookies;
 }
 
-const char* const CookieMonster::kDefaultCookieableSchemes[] = {"http", "https",
-                                                                "ws", "wss"};
+const char* const CookieMonster::kDefaultCookieableSchemes[] = {
+    "http", "https", "ws", "wss", "cyfs", "ipfs", "ipns"};
 const int CookieMonster::kDefaultCookieableSchemesCount =
     std::size(kDefaultCookieableSchemes);
 
@@ -618,8 +618,9 @@ void CookieMonster::GetAllCookies(GetAllCookiesCallback callback) {
 
   CookieList cookie_list;
   cookie_list.reserve(cookie_ptrs.size());
-  for (auto* cookie_ptr : cookie_ptrs)
+  for (auto* cookie_ptr : cookie_ptrs) {
     cookie_list.push_back(*cookie_ptr);
+  }
 
   MaybeRunCookieCallback(std::move(callback), cookie_list);
 }
