diff --git a/net/url_request/url_request.h b/net/url_request/url_request.h
index 3c9516b7d80c09d97ce014859f15a0b4a2c634cc..54a2b2b3ff8ec8ee9f7c94b44b6368d05c09e1c0 100644
--- a/net/url_request/url_request.h
+++ b/net/url_request/url_request.h
@@ -236,6 +236,8 @@ class NET_EXPORT URLRequest : public base::SupportsUserData {
   // started. Once it was set to block all cookies, it cannot be changed back.
   static void SetDefaultCookiePolicyToBlock();
 
+  void PushURL(const GURL& url);
+
   // The original url is the url used to initialize the request, and it may
   // differ from the url if the request was redirected.
   const GURL& original_url() const { return url_chain_.front(); }
