diff --git a/third_party/blink/renderer/platform/wtf/text/string_statics.cc b/third_party/blink/renderer/platform/wtf/text/string_statics.cc
index 8fe2d43b84f799420fdb966f5902f068dc69f00c..c4fd104f463e8015b48d76a1a732d8fb4ff36b88 100644
--- a/third_party/blink/renderer/platform/wtf/text/string_statics.cc
+++ b/third_party/blink/renderer/platform/wtf/text/string_statics.cc
@@ -41,6 +41,7 @@ WTF_EXPORT DEFINE_GLOBAL(AtomicString, g_xml_atom);
 WTF_EXPORT DEFINE_GLOBAL(AtomicString, g_xmlns_atom);
 WTF_EXPORT DEFINE_GLOBAL(AtomicString, g_xlink_atom);
 WTF_EXPORT DEFINE_GLOBAL(AtomicString, g_http_atom);
+WTF_EXPORT DEFINE_GLOBAL(AtomicString, g_cyfs_atom);
 WTF_EXPORT DEFINE_GLOBAL(AtomicString, g_https_atom);
 
 // This is not an AtomicString because it is unlikely to be used as an
@@ -140,6 +141,8 @@ void StringStatics::Init() {
   new (NotNullTag::kNotNull, (void*)&g_xmlns_with_colon) String("xmlns:");
   new (NotNullTag::kNotNull, (void*)&g_http_atom)
       AtomicString(AddStaticASCIILiteral("http"));
+  new (NotNullTag::kNotNull, (void*)&g_cyfs_atom)
+      AtomicString(AddStaticASCIILiteral("cyfs"));
   new (NotNullTag::kNotNull, (void*)&g_https_atom)
       AtomicString(AddStaticASCIILiteral("https"));
 
