diff --git a/third_party/blink/renderer/platform/wtf/text/atomic_string.h b/third_party/blink/renderer/platform/wtf/text/atomic_string.h
index 9a6dce8f865017d748c468552a7c557b03ec9d79..c3a2bbb753a9f2d77c731002db1c7ec5bad14d0b 100644
--- a/third_party/blink/renderer/platform/wtf/text/atomic_string.h
+++ b/third_party/blink/renderer/platform/wtf/text/atomic_string.h
@@ -282,6 +282,7 @@ WTF_EXPORT extern const AtomicString& g_star_atom;
 WTF_EXPORT extern const AtomicString& g_xml_atom;
 WTF_EXPORT extern const AtomicString& g_xmlns_atom;
 WTF_EXPORT extern const AtomicString& g_xlink_atom;
+WTF_EXPORT extern const AtomicString& g_cyfs_atom;
 WTF_EXPORT extern const AtomicString& g_http_atom;
 WTF_EXPORT extern const AtomicString& g_https_atom;
 
