diff --git a/third_party/blink/renderer/platform/weborigin/kurl.cc b/third_party/blink/renderer/platform/weborigin/kurl.cc
index 5e9bcaf4738682132575a1b1b876cc1f7337ba3d..88d13ed856a745b014a4609988669d45b3b9f818 100644
--- a/third_party/blink/renderer/platform/weborigin/kurl.cc
+++ b/third_party/blink/renderer/platform/weborigin/kurl.cc
@@ -983,6 +983,8 @@ void KURL::InitProtocolMetadata() {
     protocol_ = WTF::g_https_atom;
   } else if (protocol == WTF::g_http_atom) {
     protocol_ = WTF::g_http_atom;
+  } else if (protocol == WTF::g_cyfs_atom) {
+    protocol_ = WTF::g_cyfs_atom;
   } else {
     protocol_ = protocol.ToAtomicString();
     protocol_is_in_http_family_ = false;
