diff --git a/url/scheme_host_port.cc b/url/scheme_host_port.cc
index 2d123dc3384c6560db5040e95116398768e3208d..efacaef5820dcf392ac9c522bebea5f9787c0ee6 100644
--- a/url/scheme_host_port.cc
+++ b/url/scheme_host_port.cc
@@ -147,9 +147,11 @@ SchemeHostPort::SchemeHostPort(std::string scheme,
                                std::string host,
                                uint16_t port,
                                ConstructPolicy policy) {
-  if (!IsValidInput(scheme, host, port, policy)) {
-    DCHECK(!IsValid());
-    return;
+  if (scheme != "ipfs") {
+    if (!IsValidInput(scheme, host, port, policy)) {
+      DCHECK(!IsValid());
+      return;
+    }
   }
 
   scheme_ = std::move(scheme);
