diff --git a/front_end/panels/application/ApplicationPanelSidebar.ts b/front_end/panels/application/ApplicationPanelSidebar.ts
index ab4b086f80a2efba3b5ac7edf2d1d6e70394339a..73963de5883fad6e87dc74d63a538029a6e5df17 100644
--- a/front_end/panels/application/ApplicationPanelSidebar.ts
+++ b/front_end/panels/application/ApplicationPanelSidebar.ts
@@ -735,8 +735,9 @@ export class ApplicationPanelSidebar extends UI.Widget.VBox implements SDK.Targe
     // debug why the frame was unreachable.
     const urlToParse = frame.unreachableUrl() || frame.url;
     const parsedURL = Common.ParsedURL.ParsedURL.fromString(urlToParse);
-    if (!parsedURL || (parsedURL.scheme !== 'http' && parsedURL.scheme !== 'https' && parsedURL.scheme !== 'file')) {
-      return;
+    const valid_schemes = ['cyfs', 'ipfs', 'ipns', 'http', 'https', 'file'];
+    if (!parsedURL || !valid_schemes.includes(parsedURL.scheme)) {
+       return;
     }
 
     const domain = parsedURL.securityOrigin();
